managedkafka/snippets/clusters/list_clusters.py

```diff
@@ -12,23 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START managedkafka_list_clusters]
-from typing import List
-
-from google.cloud import managedkafka_v1
-
 
 def list_clusters(
     project_id: str,
     region: str,
-) -> List[str]:
-    """
-    List Kafka clusters in a given project ID and region.
```

Comment:
why remove this part?
Presumably it's still useful for testing the snippets?

Reply:
That's true. I re-added the docstring outside of the devrel tags.